src/amo/components/MastHead.scss

```diff
@@ -0,0 +1,66 @@
+@import "~amo/css/inc/vars";
+
+#tabzilla {
+  left: 14px;
```

Comment:
I've fixed that up to reverse the button + mozilla link, plus a couple of other fixes.